toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -112,6 +113,7 @@ var (
 	runCheck     = app.Flag("run-check", "Whether or not to run the spec file's check section during package build.").Bool()
 
 	workers          = app.Flag("workers", "Number of concurrent goroutines to parse with.").Default(defaultWorkerCount).Int()
+	concurrentNetOps = app.Flag("concurrent-net-ops", "Number of concurrent network operations to perform.").Default(defaultNetOpsCount).Int()
```

Comment:
Should we expose it as a `make` variable? No strong opinions here.

Reply:
I think we probably won't see much benefit from tuning this, going to leave it default only for now but it is an easy change if we need to adjust it later.